### PR TITLE
events: WS protobuf messages should be binary

### PR DESCRIPTION
**Pull request by swenson**
The [WebSockets spec](https://www.rfc-editor.org/rfc/rfc6455) states that text messages must be valid UTF-8 encoded strings, which protobuf messages virtually never are.

This instead correctly sends the protobuf events as binary messages.

### CODE REVIEW

File: http/events.go [hunk 1]
```diff
@@ -49,17 +49,20 @@ func handleEventsSubscribeWebsocket(args eventSubscribeArgs) (websocket.StatusCo
 		case message := <-ch:
 			logger.Debug("Sending message to websocket", "message", message)
 			var messageBytes []byte
+			var messageType websocket.MessageType
 			if args.json {
 				messageBytes, err = protojson.Marshal(message)
+				messageType = websocket.MessageText
+				messageBytes = []byte(string(messageBytes) + "\n")
```
Comment (tomhjp) [inline http/events.go:56]: OOI why the newline character? Seems like it's more a client-side concern, as the data framing is already well defined by the websocket message.

**Review by tomhjp — APPROVED** [review]
LGTM, just one off-topic question

File: http/events.go (hunk 1 above)
Comment (swenson) [reply http/events.go:56]: It's mostly for the convenience when using tools like `jq`, which work best when JSON is formatted into lines.

**Review by tomhjp — APPROVED** [review]
LGTM

**Comment by swenson** [comment]
Thanks!

**Pull request merged**